fluent-langneg/test/langneg_test.js

```diff
@@ -0,0 +1,144 @@
+
```

Comment:
Why are there two blank lines here and in `test/setup.js`? Do you know if the `use strict` pragma is still required in node? Can we skip it?